clpy/manipulation/rearrange.py

```diff
@@ -20,11 +20,11 @@ def flip(a, axis):
     """
     a_ndim = a.ndim
     if a_ndim < 1:
-        raise ValueError('Input must be >= 1-d')
+        raise core.core._AxisError('Input must be >= 1-d')
```

Comment:
Why do you modify this?

Does this cause any problem on test_rearrange.py?
If not, you should not make any changes now. This problem should be when we update the base version and merge  cupy@0d28378fe2e7176354b3b6bac71426e3f170c649.

Reply:
ValueError cause Error at 3 test cases
https://github.com/fixstars/clpy/blob/e135d7c42c9198c32cc58b93657299d54c8c52bc/tests/clpy_tests/manipulation_tests/test_rearrange.py#L179
https://github.com/fixstars/clpy/blob/e135d7c42c9198c32cc58b93657299d54c8c52bc/tests/clpy_tests/manipulation_tests/test_rearrange.py#L185
https://github.com/fixstars/clpy/blob/e135d7c42c9198c32cc58b93657299d54c8c52bc/tests/clpy_tests/manipulation_tests/test_rearrange.py#L191

So, I modify exception.
Comment outing these test case is better than this modification?

Reply:
Do you mean that the original CuPy v2.1.0 gets same error if no modification?

Reply:
yes.
I test cupy v2.1.0 and get errors at same 3 test cases.

Reply:
_Eeeeeeee...?_
We have to wait v5.0.0 to resolve this problem on CuPy...?

OK, leave that aside. Currently our target is CuPy v2.1.0. So current ClPy (v2.1.0x) should behave same as it even if it causes some errors.

Please revert the modification and modify test_rearrange.py. And make comments about this problem on the test codes and make a new issue to revert the modirication when #33 would be resolved.

Reply:
I revert rearrange.py 's modification and modify test_rearrange.py.

Reply:
I made new issue about this problem.